tests/test_run.py

```diff
@@ -37,6 +37,15 @@
     20
 ]
 
+# List of tuples containing txt file contents and the expected return value of read_data_in_txt_file()
+content_expected_pairs = [
```

Comment:
Nit: I'd suggest nesting this in `@pytest.mark.parameterize` since it doesn't seem to be used anywhere else. If you decide to keep the variable declaration, consider moving it closer to where it is actually used.